app/_src/quickstart/kubernetes.md

```diff
@@ -15,7 +15,7 @@ The zone key is purely static and arbitrary. Different zone values for different
 
 ## Prerequisites
 
-- [{{site.mesh_product_name}} installed on your Kubernetes cluster](/docs/{{ page.version }}/installation/kubernetes/)
+- {% if_version lte:2.1.x %}[{{site.mesh_product_name}} installed on your Kubernetes cluster](/docs/{{ page.version }}/installation/kubernetes/){% endif_version %}{% if_version gte:2.2.x %}[{{site.mesh_product_name}} installed on your Kubernetes cluster](/docs/{{ page.version }}/production/install-kumactl/){% endif_version %}
```

Comment:
There's a difference in all these between install kumactl and installing kuma on a k8s cluster.

This is like installing kubectl and install kubernetes or in closer terms installing deck and installing Kong

Reply:
Maybe this should be now directing to the `control plane deployment` page.

Reply:
> Maybe this should be now directing to the `control plane deployment` page.

Would that be this page? https://kuma.io/docs/2.1.x/documentation/configuration/#control-plane-configuration